tokio/src/runtime/coop.rs

```diff
@@ -52,10 +52,16 @@ impl Budget {
     ///
     /// Note that as more yield points are added in the ecosystem, this value
     /// will probably also have to be raised.
-    const fn initial() -> Budget {
+    pub(super) const fn initial() -> Budget {
         Budget(Some(128))
     }
```

Comment:
I would call this something else, because it's no longer the initial budget when you configure it.

Reply:
I would replace it with default trait